nf_core/modules/lint/main_nf.py

```diff
@@ -518,6 +536,8 @@ def _container_type(line):
     """Returns the container type of a build."""
     if re.search("bioconda::", line):
         return "bioconda"
+    if line.startswith("conda"):
```

Comment:
If you add this if, which I think it's a good change :) You can remove the previous one using "bioconda", as they are equivalent, our container declaration for conda looks like this `conda "bioconda::bcftools=1.16"`

Reply:
ok i have removed the bioconda return to the function. i knew there was overlap between the two but i wasn't sure whether i should touch it.